omod/src/test/java/org/openmrs/module/referenceapplication/page/controller/LoginPageControllerTest.java

```diff
@@ -596,10 +614,29 @@ public void post_shouldAutoSelectALocationAfterAuthenticationIfLoginLocationsSiz
 		when(administrationService.getGlobalProperty(eq(ReferenceApplicationConstants.LOCATION_USER_PROPERTY_NAME)))
 		        .thenReturn("someValue");
 		
-		assertEquals(homeRedirect, new LoginPageController().post(USERNAME, PASSWORD, null, locationService,
-		    administrationService, uiUtils, appFrameworkService, pageRequest, sessionContext));
+		assertEquals(homeRedirect, new LoginPageController().post(USERNAME, PASSWORD, null, null, locationService,
+		    administrationService, uiUtilsWithoutTimezones, appFrameworkService, pageRequest, sessionContext));
 		
 		verify(sessionContext, times(1)).setSessionLocation(eq(location));
 	}
-	
+
+	@Test
+	public void post_shouldNotAutoSelectALocationAfterAuthenticationIfLoginLocationsSizeIsOneAndUsingTimezonesWithoutClientTimezone() throws Exception {
+		when(Context.isAuthenticated()).thenReturn(false).thenReturn(true);
+		when(Context.getAuthenticatedUser()).thenReturn(mock(User.class));
+		final int locationId = 1;
```

Comment:
Did you look at our naming convention for constants? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
I based that name on the example above "post_shouldAutoSelectALocationAfterAuthenticationIfLoginLocationsSizeIsOne".

Reply:
@icrc-fdeniger i mean the convention for naming constants.

Reply:
I'll ping Joel for that point :)

Reply:
@icrc-fdeniger loool thanks 😊